fact_store/model.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, Float, Boolean, Sequence
+
+
+Base = declarative_base()
+
+
+class EventModel(Base):
+    """
+    Model Used for persisting events when model generates a prediction.
+    We track this for quality control
+    """
+    __tablename__ = "events"
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    message = Column(String(255), nullable=False)
+    predict_id = Column(String(255), nullable=False, unique=True)
+    score = Column(Float,  nullable=False)
+    anomaly_status = Column(Boolean,  nullable=False)
+
+
+    def __repr__(self):
+        return '<Event %r>' % self.message
+
+    def to_dict(self):
+        return {c.name: getattr(self, c.name) for c in self.__table__.columns}
+
+
+class FeedbackModel(Base):
+    """FeedbackModel
+    Persists user feedback to track false predictions that they found.
+    We will use this data
+    for our model to continously learn.
+    """
+    __tablename__ = "feedback"
+    id = Column(Integer,
+                Sequence('id_seq'),
+                primary_key=True,
+                autoincrement=True)
+    # Required
+    predict_id = Column(String(255),
```

Comment:
Should the `predict_id` be an FK for Event `predict_id`?

Reply:
Yes it should.

Reply:
Yes it should be a foriegn key.

Reply:
Fixed